rubygem-hammer_cli_gutterball.spec

```diff
@@ -50,7 +49,8 @@ gem install --local --install-dir .%{gem_dir} \
 
 %install
 mkdir -p %{buildroot}%{_sysconfdir}/%{confdir}/cli.modules.d
-install -m 755 %{SOURCE1} %{buildroot}%{_sysconfdir}/%{confdir}/cli.modules.d/gutterball.yml
+install -m 755 .%{geminstdir}/config/gutterball.yml %{buildroot}%{_sysconfdir}/%{confdir}/cli.modules.d/gutterball.yml
```

Comment:
Yea, i was noticing that, I suppose I can minimalize the change here and then do the standardization on the k-packing PR.